dll/cpl/hotplug/hotplug.c

```diff
@@ -372,6 +374,16 @@ SafeRemovalDlgProc(
                     SetWindowTextW(hwndDlg, szWindowTitle);
                 }
 
+                hIcon = LoadIconW(hApplet, MAKEINTRESOURCEW(IDI_HOTPLUG));
```

Comment:
It is better to use LoadImage here as well.

Reply:
OK.